politeiad/backendv2/tstorebe/plugins/comments/cmds.go

```diff
@@ -570,11 +567,8 @@ func (p *commentsPlugin) cmdEdit(token []byte, payload string) (string, error) {
 	}
 
 	// Ensure no extra data provided if not allowed
-	if !p.allowExtraData && (e.ExtraData != "" || e.ExtraDataHint != "") {
-		return "", backend.PluginError{
-			PluginID:  comments.PluginID,
-			ErrorCode: uint32(comments.ErrorCodeExtraDataNotAllowed),
-		}
+	if err = p.validateExtraData(e.ExtraData, e.ExtraDataHint); err != nil {
```

Comment:
`validate` and `verify` are often used interchangeably.  It doesn't really matter which one you use within a repo as long as the repo is consistent.  I chose to use `verify` in this repo because it's shorter and shorter function names are easier to read.  The naming convention used here should match the rest of the repo.